src/classes/UTIL_Describe.cls

```diff
@@ -202,6 +206,7 @@
     * if the field is not accessible
     * @param objectName the name of the object the field belongs to
     * @param fieldNames the set of field names to check read access
+    * @return void
     ********************************************************************************************************/
     public static void checkFieldReadAccess(String objectName, String fieldName) {
         fieldName = fieldName.contains('__c') ? StrTokenNSPrefix(fieldName) : fieldName;
```

Comment:
@davidhabib I don't think this will work with fieldsets because the field from the fieldset will already have the namespace right?